packages/upswyng-web/src/components/Resource.tsx

```diff
@@ -81,7 +86,11 @@ export const Resource = () => {
 
   return (
     <Container>
-      <PageBanner color={resourceColor} text={resource.name} />
+      <PageBanner
+        color={resourceColor}
+        text={resource.name}
+        backRef={!!lastLocation ? history.goBack : undefined}
```

Comment:
I vary rarely (never?) intentionally set something to `undefined`.  I would pass `null` here.. you might have to change the type of the prop to `() => void | null`.. not 100% sure.